src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,41 +1,63 @@
 package techcourse.myblog.domain;
 
-import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import techcourse.myblog.domain.exception.InvalidAccessException;
 
 import javax.persistence.*;
 
-@Builder
-@Getter
-@Setter
-@NoArgsConstructor
 @Entity
+@NoArgsConstructor
+@EqualsAndHashCode
+@Getter
+@Table(name = "article")
 public class Article {
+    private static final String INVALID_ERROR = "작성자가 아닙니다.";
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private int id;
+    @Column(name = "id")
+    private long id;
 
-    @Column(name = "title", length = 500, nullable = false)
+    @Column(name = "title", nullable = false)
     private String title;
 
-    @Column(name = "contents", length = 6000, nullable = false, columnDefinition = "TEXT")
+    @Column(name = "coverUrl", nullable = false)
+    private String coverUrl;
+
+    @Lob
+    @Column(name = "contents", nullable = false)
     private String contents;
 
-    @Column(name = "coverUrl", length = 1000, nullable = false)
-    private String coverUrl;
+    @ManyToOne
+    @OnDelete(action = OnDeleteAction.CASCADE)
+    @JoinColumn(name = "author", foreignKey = @ForeignKey(name = "fk_article_to_user"), nullable = false)
+    private User author;
 
-    public Article(int id, String title, String contents, String coverUrl) {
-        this.id = id;
+    public Article(String title, String coverUrl, String contents) {
         this.title = title;
-        this.contents = contents;
         this.coverUrl = coverUrl;
+        this.contents = contents;
     }
 
-    public void updateArticle(String title, String coverUrl, String contents) {
-        this.title = title;
-        this.coverUrl = coverUrl;
-        this.contents = contents;
+    public void update(Article articleToUpdate, long loginUserID) {
```

Comment:
아래의 Google Java Style Guide를 참고해 주세요.
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

```suggestion
    public void update(Article articleToUpdate, long loginUserId) {
```

Reply:
수정했습니다!